corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -264,3 +265,28 @@ def delete_diff_db(domain):
         # reraise on any other error
         if e.errno != 2:
             raise
+
+
+BASE_IGNORED_FORM_PATHS = {
+    '_rev',
+    'migrating_blobs_from_couch',
+    '#export_tag',
+    'computed_',
```

Comment:
haha forgot about this thing. we should :skull: sometime (once MVP reports are also gone)